code/game/machinery/computer/cloning.dm

```diff
@@ -179,6 +179,23 @@
 	var/dat = ""
 	dat += "<a href='byond://?src=[REF(src)];refresh=1'>Refresh</a>"
 
+	if(pods)
+		for(var/P in pods)
+			var/obj/machinery/clonepod/pod = P
+			if(pod.efficiency >= 3)
```

Comment:
We need a case that detects if it's 1 or 2, since if efficiency were to be set lower than 3 after previously being at/above 3, the price won't change back to 100, which is how it starts.

Reply:
Thanks for that!